workflow/rules/authentic.smk

```diff
@@ -0,0 +1,225 @@
+# the checkpoint that shall trigger re-evaluation of the DAG
+checkpoint Extract_TaxIDs:
+    input:
+        pathogens="results/KRAKENUNIQ/{sample}/taxID.pathogens",
+    output:
+        dir=directory("results/AUTHENTICATION/{sample}"),
+    shell:
+        "mkdir -p {output.dir}; "
+        "while read taxid; do mkdir {output.dir}/$taxid; done<{input.pathogens}"
+
+
+def aggregate_PMD(wildcards):
```

Comment:
Sounds super useful, I'll take a look. Are all python functions inside `common.smk`? And is that the place you think these should go?

Reply:
That's what the best practice recommends. In case common.smk becomes too large one could consider splitting it up (I occasionally do this) but for now I think we can put everything in common.smk.

Reply:
pre-commit can feel a bit intrusive to begin with, but I really like how it does linting/formatting early on.